styles.css

```
@@ -72,7 +95,9 @@
 .agentic-chat-message {
   border-radius: var(--radius-m);
   max-width: 100%;
+  min-width: 0;
   word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)